skills/lark-shared/references/lark-shared-identity-and-permissions.md

```
@@ -9,6 +9,7 @@
 | 指定单个 scope 授权 | `lark-cli auth login --scope "<scope>" --no-wait --json` |
 | 检查当前登录态、是谁登录、token 是否有效 | `lark-cli auth status --json --verify`；回答时引用 `identity`、`verified`、`identities.user.status`、`identities.user.userName`、`identities.user.openId`（用户 open id）、`identities.user.tokenStatus`、`identities.user.scope` |
 | 快速查看当前身份状态 | `lark-cli whoami`；实际生效的那一个身份 |
+| 从外部凭据源安全注入 bot TAT | `<token-provider> | lark-cli auth import-tenant-token --app-id <cli_xxx> --token-stdin`；禁止把 token 放进 argv |
```

markdownlint-cli2 (0.23.2)

[warning] Table column count
Expected: 2; Actual: 3; Too many cells, extra data will be missing

(MD056, table-column-count)